saos-restful-web-api/src/main/java/pl/edu/icm/saos/api/formatter/LawJournalEntryCodeFormat.java

```diff
@@ -0,0 +1,19 @@
+package pl.edu.icm.saos.api.formatter;
```

Comment:
I'm not sure if it should be a part of api. Wouldn't we like to use it on webapp alone too?